toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -91,6 +96,47 @@ func buildDracutModule(dracutModuleName string, dracutDriverName string, imageCh
 	return nil
 }
 
+func updateFstab(buildDir string, imageChroot *safechroot.Chroot, imageConnection *ImageConnection) error {
+	var err error
+
+	diskPartitions, err := diskutils.GetDiskPartitions(imageConnection.Loopback().DevicePath())
+	if err != nil {
+		return err
+	}
+
+	rootfsPartition, err := findRootfsPartition(buildDir, diskPartitions)
+	if err != nil {
+		return err
+	}
+
+	fstabFile := filepath.Join(imageChroot.RootDir(), "etc", "fstab")
+	lines, err := file.ReadLines(fstabFile)
+	if err != nil {
+		return fmt.Errorf("failed to read fstab file: %v", err)
+	}
+
+	var updatedLines []string
+	fstabLineRegex := regexp.MustCompile(`^PARTUUID=` + rootfsPartition.PartUuid + ` .*`)
```

Comment:
The user can specify which mount identifier type the partition uses. So, you can't assume `PARTUUID` will be used.

Reply:
I would look for the entry that mounts to `/`.

Reply:
Fixed.